src/System.Windows.Forms/src/System/Windows/Forms/DataGridView.Methods.cs

```diff
@@ -11692,9 +11724,10 @@ protected virtual void OnAutoSizeColumnsModeChanged(DataGridViewAutoSizeColumnsM
                             dataGridViewColumn.CachedThickness = dataGridViewColumn.Thickness;
                         }
 
-                        AutoResizeColumnInternal(dataGridViewColumn.Index,
-                                                 (DataGridViewAutoSizeColumnCriteriaInternal)autoSizeColumnMode,
-                                                 (((DataGridViewAutoSizeRowsModeInternal)_autoSizeRowsMode) & DataGridViewAutoSizeRowsModeInternal.AllColumns) == 0 /*fixedHeight*/);
+                        AutoResizeColumnInternal(
+                            dataGridViewColumn.Index,
+                            (DataGridViewAutoSizeColumnCriteriaInternal)autoSizeColumnMode,
+                            (((DataGridViewAutoSizeRowsModeInternal)_autoSizeRowsMode) & DataGridViewAutoSizeRowsModeInternal.AllColumns) == 0 /*fixedHeight*/);
```

Comment:
Looks like this was missed
```suggestion
                        AutoResizeColumnInternal(
                            dataGridViewColumn.Index,
                            (DataGridViewAutoSizeColumnCriteriaInternal)autoSizeColumnMode,
                            fixedHeight: (((DataGridViewAutoSizeRowsModeInternal)_autoSizeRowsMode) & DataGridViewAutoSizeRowsModeInternal.AllColumns) == 0);
```
Could we actually use `HasFlags()` for input for `fixedHeight`?

Reply:
I changing to use `HasFlags()` here was missed, but I won't block on this. Could you address this in your next change?

Reply:
We have many usages of the bitwise operators instead of `HasFlag` in the codebase. If you want to replace them, perhaps open a separate issue? As far as I remember though, this is not guaranteed to be optimized as per https://github.com/dotnet/runtime/issues/55455#issuecomment-877639565, so not sure if we should keep doing that.